src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -165,25 +168,29 @@ public void progress(ProgressMessage message) throws DockerException {
   private static final long DEFAULT_READ_TIMEOUT_MILLIS = SECONDS.toMillis(30);
   private static final int DEFAULT_CONNECTION_POOL_SIZE = 100;
 
-  private static final ClientConfig DEFAULT_CONFIG = new ClientConfig(
-      ObjectMapperProvider.class,
-      JacksonFeature.class,
-      LogsResponseReader.class,
-      ProgressResponseReader.class);
+  private static final ClientConfig DEFAULT_CONFIG =
```

Comment:
Perhaps we can revert these code style changes? Looks like it might've been done by your IDE automatically.